docs/wiki/PipelinesDesign.md

```diff
@@ -22,11 +26,53 @@ To "build"/"bake" the modules, a dedicated pipeline is used for each module to v
 
 Using this flow, validated modules can be consumed by other any consumer / template / orchestration to deploy a workload, solution, environment or landing zone.
 
-## Validation prerequisites
+## Validate
+
+### Validation prerequisites
 
 A _"Sandbox"_ or _"Engineering"_ **validation subscription** (in Azure) has to be used to test if the modules (or other components) are deployable. This subscription must not have connectivity to any on-premises or other Azure networks.
 The module validation pipelines use an Azure Active Directory Service Principal (AAD SPN) to authenticate to the validation subscription and run the test deployments of the modules.
 
-## Why do I have to validate deployments of modules?
+### Why do I have to validate deployments of modules?
 
 Since every customer environment might be different due to applied Azure Policies or security policies, modules might behave differently or naming conventions need to be tested and applied beforehand.
+
+### Tokens Replacement
+
+The [validation composite action (pipeline)](../../.github/actions/templates/validateModuleApis/action.yml) includes a step that replaces certain strings in a parameter file with values that are provided from the module workflow. This helps achieve the following:
```

Comment:
Relative links only works when its relative within the docs folder when this is published into the wiki. Use public link.

Reply:
This I feel is going to be a trial and error for me to get it right. I've fixed the issue and even referenced images like other images (using HTML).